### ISSUE TITLE
Window drag and drop doesn't work if you click on the window title and is completely impossible on long titles in macOS.

### ISSUE THREAD

**Issue by pilgunboris** [brave/brave-browser#43046]
### Description

Hello, 

I'm using: 

 - macOS 15.2 (24C101)  (but this issue also existed in previous versions.)

 - Brave is up to date Version 1.73.104 Chromium: 131.0.6778.204 (Official Build) (arm64)

If I try dragging the window around the screen I can do this only when I avoid the highlighted zone with the text. In the case of this example, the title is short so I can easily click on the blue area around it and drag the window, however, with the long page titles it is almost impossible and is super annoying when I try to use tiling to work with 2 or more windows at the same time on my 16" screen.

<img width="1566" alt="Image" src="https://github.com/user-attachments/assets/69940087-013e-4997-9b6f-a5c083cbcf04" />

### Steps to reproduce

1. Open any website with a long page title caption (find an article with a very long title that covers the whole window title from side to side). Alternatively, use shorter text but reduce the window size so that the text covers all window title space.
2. Try dragging the window by clicking on the text in the window title
3. Confirm that the window does not react like it normally does if you click outside of the text area in the window title area.

### Actual result

The window does not react if you click on the text.

### Expected result

The window is draggable just like normally when I click at the blue area around the window title text.

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Version 1.73.104 Chromium: 131.0.6778.204 (Official Build) (arm64)

### Channel information

- [x] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [x] with Brave Shields disabled
- [x] with Brave Rewards disabled
- [x] in the latest version of Chrome

**Comment by rebron** [comment]
@pilgunboris You're using vertical tabs correct?

**Comment by pilgunboris** [comment]
Yes, that's right. Here are all the apperance settings just in case:

<img width="1033" alt="Image" src="https://github.com/user-attachments/assets/8504bfef-67e5-45ef-a3d7-0fe40671488d" />
<img width="1071" alt="Image" src="https://github.com/user-attachments/assets/69cea9e9-7f33-43f9-a588-f4847a9b0ddc" />

**Comment by ellmann-creative** [comment]
Can confirm behaviour on macOS Ventura (13.7).

**Comment by Raphmatt** [comment]
Any update on this? This issue is impacting my workflow, on macOS I need to carefully aim between menu items (close, minimize, fullscreen buttons) to drag Brave windows. Any ETA or temporary workaround would be appreciated. Thanks!

<img width="1206" height="197" alt="Image" src="https://github.com/user-attachments/assets/2e0234dd-ab92-4244-8fc6-af660b553ca0" />
